tests/testthat/test-int-latent_individual.R

```diff
@@ -68,6 +68,74 @@ test_that("epidist.epidist_latent_individual Stan code has no syntax errors and
   expect_no_error(mod_gamma$compile())
 })
 
+test_that("epidist.epidist_latent_individual fits and the MCMC converges  in the gamma delay case", { # nolint: line_length_linter.
+  skip_on_cran()
+  set.seed(1)
+  
+  gamma_family <- epidist_family(
+    prep_obs_gamma,
+    family = "gamma",
+    dpars = c("mu", "shape"),
+    links = c("log", "log"),
+    lb = c(0, 0),
+    ub = c(NA, NA)
+  )
+
+  gamma_formula <- epidist_formula(prep_obs_gamma)
```

Comment:
Its shape and rate right?

Reply:
<img width="772" alt="image" src="https://github.com/user-attachments/assets/2436c831-ecb9-406a-8de9-be3bf2546769">

See https://cran.r-project.org/web/packages/brms/vignettes/brms_families.html. I think it's `mu` and `shape`.

Reply:
ah right its mu and shape it seems: https://solomonkurz.netlify.app/blog/2023-05-14-causal-inference-with-gamma-regression-or-the-problem-is-with-the-link-function-not-the-likelihood/

Reply:
Yes I'm confused why people are using `identity` link for `mu` when it's got to be positive (?)

Reply:
I don't think they are I think they are doing that for shape? See comment about running some raw brms

Reply:
You don't think they are using `indentity` for `shape`? But you also write "It looks like it expects the shape to have no link" -- isn't that the same as identity link (or can be constructed to be the same).

And they are using `identity` for `mu` (sometimes):

<img width="398" alt="image" src="https://github.com/user-attachments/assets/54951098-5759-459c-830f-c606cf134761">